src/js/pensions/components/IntroductionPage.jsx

```diff
@@ -52,10 +52,11 @@ class IntroductionPage extends React.Component {
               </div>
               <br/>
               <h6>Aid and Attendance and Housebound benefits</h6>
-              If you’re housebound, you may qualify for Aid and Attendance and Housebound benefits in addition to your pension benefits.
+              If you need help with your daily living activities, or if you’re confined to your home, you may qualify for an Aid and Attendance (A&A) or Housebound allowance in addition to your pension benefits.<br/>
+              <a href="/pension/aid-attendance-housebound/">Learn more about eligibility</a>.<br/><br/>
```

Comment:
Should we open link in new tabs using target="_blank", like you've done  in `src/js/pensions/helpers.jsx` below?